posthog-core/src/index.ts

```diff
@@ -1755,25 +1754,24 @@ export abstract class PostHogCore extends PostHogCoreStateless {
             }
           }
           this.setKnownFeatureFlagDetails(newFeatureFlagDetails)
-          // Mark that we hit the /decide endpoint so we can capture this in the $feature_flag_called event
-          this.setPersistedProperty(PostHogPersistedProperty.DecideEndpointWasHit, true)
-
-          this.cacheSessionReplay('decide/flags', res)
+          // Mark that we hit the /flags endpoint so we can capture this in the $feature_flag_called event
+          this.setPersistedProperty(PostHogPersistedProperty.FlagsEndpointWasHit, true)
+          this.cacheSessionReplay('flags', res)
```

Comment:
I deprecated this property.  Is that risky?  I could migrate it instead and have some sort of method that reads both fields, but I'm also fine just ripping the bandaid.